source/agora/consensus/data/ConsensusParams.d

```diff
@@ -67,7 +72,8 @@ public immutable class ConsensusParams
                  uint max_quorum_nodes = 7, uint quorum_threshold = 80,
                  uint quorum_shuffle_interval = 30,
                  time_t genesis_start_time = 1596179709,
-                 uint block_interval_sec = 1)
+                 uint block_interval_sec = 1,
+                 const ref PublicKey commons_budget_address = CommonsBudgetKeyPair.address)
```

Comment:
Second parameter, `in`, no  default value

Reply:
The default value should be passed  at L94

Reply:
I fixed the code.